addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -217,54 +218,24 @@ def _l10n_it_post_invoices_step_2(self, invoices):
                 to_return[invoice] = response
                 continue
 
-            state = response['state']
+            # In some transaction states we don't need to read the attachment
+            state = response.get('state')
```

Comment:
Why?  Is there a case where the response does not have a state?

Reply:
Guaps... no.